Makefile

```diff
@@ -61,6 +61,37 @@ ORIG_FILE_URL = "$(SOURCE_FILE_BASE_URL)/$(ORIG_FILE)"
 NON_UP_DIR = /tmp/non_upstream_patches
 
 $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# Include any non upstream patches
+	rm -rf $(NON_UP_DIR)
+	mkdir -p $(NON_UP_DIR)
+
+	if [ ! -z ${EXTERNAL_KERNEL_PATCH_URL} ];  then
+		wget $(EXTERNAL_KERNEL_PATCH_URL) -O patches.tar
+		tar -xf patches.tar -C $(NON_UP_DIR)
+	fi
+
+	# Precedence is given for external URL
+	if [ -z ${EXTERNAL_KERNEL_PATCH_URL} ] && [ x${INCLUDE_EXTERNAL_PATCHES} == xy ]; then
```

Comment:
I think we had a discussion on the variable naming in #296, but since this is being renamed to `INCLUDE_EXTERNAL_PATCHES`, can `EXTERNAL_KERNEL_PATCH_URL` fall under that, such that if `INCLUDE_EXTERNAL_PATCHES` is set to `n`, then nothing happens, even if `EXTERNAL_KERNEL_PATCH_URL` is specified?

Reply:
That means user has to provide both `EXTERNAL_KERNEL_PATCH_URL  `& `INCLUDE_EXTERNAL_PATCHES `, if they wish to use non-upstream patches from an external URL.  Why have two control options?  `INCLUDE_EXTERNAL_PATCHES` is for locally saved non-up patches. atleast that's how i thought of

Reply:
@saiarcot895 Do you think it makes more sense to make `INCLUDE_EXTERNAL_PATCHES` a mandatory parameter.

Reply:
My thought was that because of the name `INCLUDE_EXTERNAL_PATCHES`, it is a parameter whether to include external patches at all, and that `EXTERNAL_KERNEL_PATCH_URL` is one source of those external patches.

Reply:
Hmm, will update it then

Reply:
Handled